certified-connectors/Azure Communication Services Chat/apiProperties.json

```diff
@@ -0,0 +1,24 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Endpoint URL",
+          "description": "Azure Communication Services endpoint URL.",
+          "tooltip": "Find your endpoint URL in your Azure Communication Services resource in Azure Portal in the \"Overview\" tab.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@lei-msft default icon color is [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), can you please change it?